goroutine/gid_tinygo.go

```diff
@@ -0,0 +1,37 @@
+//go:build tinygo
+
+package goroutine
+
+import (
+	"sync"
+	"unsafe"
+)
+
+const IsTinyGo = true
+
+var (
+	counter int64
+	mapping = make(map[uintptr]int64)
+	// TinyGo at the moment is single threaded so this is not needed but it's good to have anyway
+	// in case that changes. It does had ~5ns (from 20ns vs 4ns big go) but it's better to be correct.
```

Comment:
```suggestion
	// in case that changes. It does add ~5ns (from 20ns vs 4ns big go) but it's better to be correct.
```

`does had` is grammatically wrong, I think it's `add`